lib/logstash/inputs/rabbitmq/march_hare.rb

```diff
@@ -112,6 +112,9 @@ def consume
       @consumer = @q.build_consumer(:block => true) do |metadata, data|
         @codec.decode(data) do |event|
           decorate(event)
+	  metadata.headers.each do |header, value|
+            event[header] = value.respond_to?('toString') ? value.toString : value 
+         end if @include_headers
```

Comment:
Concerning the typecasting here, I am not very familiar with `metadata.headers` method can it return something else than primitives? (String/Integer.. )